Tools/macOS/Nightlies.plist.in

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file for?

Reply:
It's a [launchd](https://kapeli.com/dash_share?docset_file=Man_Pages&page=launchd&section=8) script, that is being used to set up a cronjob on macOS. If and when we can go on GitHub Action, we wouldn't need this, but for now, I have it to set up a nightly on either the lab computer or my Mac mini.

Reply:
We can still run a script as well right?
Because then I could plug it into the buildbot (and maybe try to get that working properly again?)
Although I would prefer to not spend time on that and have everything on actions...

Reply:
Yes, you can still do that. I hope we can put it on GitHub as well, but at the moment, we really cannot.